site/content/blog/2023-12-18-user-personas-poll/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Discovering Our Users - The ZAP User Personas Questionnaire"
+summary: >
+  Improving usability requires the identification of the type of users ZAP has
+images:
+- https://www.zaproxy.org/blog/2023-12-18-user-personas-poll/535a4e71-d6a6-40a1-8289-f31e8810b65b.png
+type: post
+tags:
+- blog
+- update
+date: "2023-12-19"
+authors:
+- yiannis
+---
+
+## Discovering Our Users - The ZAP User Personas Questionnaire
```

Comment:
This one is redundant, the blog template already includes the title above the content.